src/include/network/lan/complex.rb

```diff
@@ -515,7 +575,9 @@ def input_done?(ret)
       true
     end
 
-    def MainDialog(init_tab)
+    def MainDialog(init_tab, builder: nil)
```

Comment:
and again what nil means here? New item and new builder?

Reply:
nothing. Should be ```(builder:)``` and mandatory

Reply:
replaced